test/tests/restart/restart_subapp_not_master/two_step_solve_sub_restart.i

```diff
@@ -0,0 +1,82 @@
+[Mesh]
+  file = two_step_solve_master_cp/0002_mesh.cpr
+[]
+
+[Problem]
+  restart_file_base = two_step_solve_master_cp/LATEST
+  allow_sub_app_restart = true
```

Comment:
No, the problem is that the header doesn't get read due to the use of
isUltimateMaster() in a few key spots. This was causing the set fault which
I fixed in this PR as well. I need to have the header read which is what
the flag accomplishes in the sub app case.
On Tue, Mar 8, 2016 at 4:38 PM Derek Gaston <notifications@github.com>
wrote:

> In
> test/tests/restart/restart_subapp_not_master/two_step_solve_sub_restart.i
> <https://github.com/idaholab/moose/pull/6531#discussion_r55448742>:
>
> > @@ -0,0 +1,82 @@
> > +[Mesh]
> > +  file = two_step_solve_master_cp/0002_mesh.cpr
> > +[]
> > +
> > +[Problem]
> > +  restart_file_base = two_step_solve_master_cp/LATEST
> > +  allow_sub_app_restart = true
>
> Is it really necessary to allow_sub_app_restart? Shouldn't it be the case
> that just specifying restart_file_base in a sub_app should be enough?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6531/files#r55448742>.
>